### PR TITLE
[Devops] Adding initial dockerfile

### PR DESCRIPTION
**Pull request by rafaiovlabs**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## relay-dapp dockerfile (This file is called Dockerfile) on server side. ##
+FROM node:12-alpine AS compiler
+RUN apk add --no-cache build-base git bash
+WORKDIR /usr/src/app
+COPY package*.json ./
+COPY .env ./.env
```
Comment (antomor) [inline Dockerfile]: Why do we need to specify the destination filename?
I mean `COPY .env ./` is enough, isn't it?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+RUN npm i
```
Comment (antomor) [inline Dockerfile]: Why this `npm i` is interleaved between the copy steps?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+COPY . ./
+RUN npm start
```
Comment (antomor) [inline Dockerfile:7]: Should we adopt a multi-stage build as we did for the server?

**Review by antomor — COMMENTED** [review]
I just left some doubts, maybe due to my superficial knowledge of docker

File: Dockerfile (hunk 1 above)
Comment (rafaiovlabs) [reply Dockerfile]: Changed thanks

File: Dockerfile (hunk 2 above)
Comment (rafaiovlabs) [reply Dockerfile]: Changed thanks.

File: Dockerfile (hunk 3 above)
Comment (rafaiovlabs) [reply Dockerfile:7]: As this application is fairly simple for now we can leave it as it is.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+## relay-dapp dockerfile (This file is called Dockerfile) on server side. ##
+FROM node:12-alpine AS compiler
+RUN apk add --no-cache build-base git bash
+WORKDIR /usr/src/app
+COPY package*.json ./
+COPY .env ./
+COPY . ./
```
Comment (antomor) [inline Dockerfile]: sorry but with the copy command we run on line #7, do we still need lines #5 and #6?

**Review by antomor — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 4 above)
Comment (rafaiovlabs) [reply Dockerfile]: Sorry thats correct, changed.

**Review by antomor — APPROVED** [review]
LGTM

**Pull request merged**